app/metrics.yaml

```diff
@@ -6761,6 +6761,45 @@ autoplay:
       tags:
         - SitePermissions
 
+cookie_banners:
+  visited_setting:
+    type: event
+    description: A user visited the cookie banner handling screen
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1796146
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/27561
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: 118
+    metadata:
+      tags:
+        - Privacy&Security
+  setting_changed:
+    type: event
+    description: |
+      A user changed their setting.
+    extra_keys:
+      cookie_banner_setting:
+        description: |
+          The new setting for autoplay: disabled,reject_all,
```

Comment:
The new settings for cookie banner handling : disabled or reject_or_accept_all.

Reply:
I added it as part of the telemetry as the UI could change in the future and it could be a possible value.

Reply:
Ok , but the description is " The new setting for autoplay" . I think it should be "The new settings for cookie banner handling "

Reply:
👍🏽

Reply:
Updated!